twilight-http/src/request/guild/update_guild.rs

```diff
@@ -165,7 +165,15 @@ impl<'a> UpdateGuild<'a> {
     /// Attempting to add or remove the `COMMUNITY` feature requires the
```

Comment:
we could link to the `GuildFeature` enum here

Reply:
Should I retarget the PR to `next` and also change the functions arguments to take the enum?

Reply:
that can be another pr, the enum already works as an input